lib/livebook_web/components/core_components.ex

```diff
@@ -449,12 +449,13 @@ defmodule LivebookWeb.CoreComponents do
   attr :source_id, :string, required: true
   attr :language, :string, required: true
   attr :source, :string, required: true
+  attr :wrap, :boolean, default: false
 
   def code_preview(assigns) do
     ~H"""
     <div class="markdown">
       <pre><code
-      class="tiny-scrollbar"
+      class={@wrap && "break-all whitespace-pre-wrap"}
```

Comment:
We still want the scrollbar by default

```suggestion
      class={if(@wrap, do: "break-all whitespace-pre-wrap", else: "tiny-scrollbar")}
```